frontend/src/scenes/insights/ActionFilter/entityFilterLogic.test.ts

```diff
@@ -55,26 +55,36 @@ describe('entityFilterLogic', () => {
         })
     })
 
-    it('renames an entity with a custom name', async () => {
+    describe('renaming filters', () => {
         const filterWithCustomName = {
             id: '$pageview',
             name: '$pageview',
             custom_name: 'Custom event name',
             order: 0,
         }
 
-        await expectLogic(logic, () => {
-            logic.actions.renameFilter(filterWithCustomName)
-        }).toDispatchActions(['renameFilter', 'updateFilter', 'setFilters'])
+        it('renames successfully', async () => {
+            await expectLogic(logic, () => {
+                logic.actions.renameFilter(filterWithCustomName)
+            }).toDispatchActions(['renameFilter', 'updateFilter', 'setFilters'])
 
-        expect(logic.props.setFilters).toBeCalledWith(
-            expect.objectContaining({
-                events: expect.arrayContaining([
-                    expect.objectContaining({
-                        custom_name: filterWithCustomName.custom_name,
-                    }),
-                ]),
+            expect(logic.props.setFilters).toBeCalledWith(
+                expect.objectContaining({
+                    events: expect.arrayContaining([
+                        expect.objectContaining({
+                            custom_name: filterWithCustomName.custom_name,
+                        }),
+                    ]),
+                })
+            )
+        })
+
+        it('close modal after renaming', () => {
+            expectLogic(logic, () => {
+                logic.actions.renameFilter(filterWithCustomName)
             })
-        )
+                .toDispatchActions(['renameFilter', 'setModalVisible'])
```

Comment:
I like to call my actions with slightly more explicit names, such as `showModal`, `hideModal` and avoid the `set*` pattern as much as possible. Making that change would also make it easier to test: 

```ts
.toDispatchActions(['renameFilter', 'hideModal'])
```

However, you can also explicitly test that the modal was hidden with:

```ts
.toDispatchActions(['renameFilter', logic.actionCreators.setModalVisible(false)])
```

Reply:
Sweet, didn't know I could pass in an actionCreator w/ parameters here as well.